### PR TITLE
add pss exceptions for remaining phoenix components

### PR DESCRIPTION
**Pull request by bdehri**
### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

**Comment by stone-z** [comment]
Hey @bdehri we have some existing policy exceptions here as well and would like to keep the naming, labeling, and gating consistent. Would recommend https://github.com/giantswarm/kyverno-app/pull/290 for an example

**Comment by bdehri** [comment]
@stone-z Could you please take a look again?

File: helm/kyverno/templates/core-policies/azure-scheduled-events-polex.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{{- if .Values.policyExceptions.enableAwsEbsCsiDriverPolex }}
```
Comment (stone-z) [inline helm/kyverno/templates/core-policies/azure-scheduled-events-polex.yaml:1]:
```suggestion
{{- if .Values.policyExceptions.enableAzureScheduledEventsPolex }}
```

Comment (stone-z) [reply helm/kyverno/templates/core-policies/azure-scheduled-events-polex.yaml:1]: Is `azure-scheduled-events` actually needed prior to Kyverno during cluster creation?

Comment (bdehri) [reply helm/kyverno/templates/core-policies/azure-scheduled-events-polex.yaml]: at the second look, it is not.

**Review by stone-z — APPROVED** [review]

**Pull request merged**